Scripts~/windows/.verify-working-directory.ps1

```diff
@@ -0,0 +1,10 @@
+$ROOT_DIR="."
+$RUNTIME_PATH="$ROOT_DIR\Runtime"
+
+if (-Not (Test-Path -Path $RUNTIME_PATH))
```

Comment:
You might be able to remove the need for this file at all if you use "$PSScriptRoot" instead of "." as "$ROOT_DIR".

Reply:
Removed